src/components/Logo/AnimatedLogo.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import s from './styles.module.scss';
+
+export const AnimatedLogo = props => {
+  const { classnames } = props;
```

Comment:
you should import `classnames` library, not pass it through props

Reply:
Will do